packages/openneuro-components/src/search-page/SearchResults.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { Story, Meta } from '@storybook/react'
+import { mri } from '../mock-content/mri-search-results'
+
+import { SearchResult } from './SearchResult'
+
+export default {
+  title: 'Components/SearchResults',
+  component: SearchResult,
+} as Meta
+
+const SearchResultsTemplate: Story = ({ items, profile }) => {
+  console.log(items)
+  return (
+    <div className="search-results">
+      {items.map((item, index) => (
+        <SearchResult node={items[index].node} profile={profile} />
```

Comment:
Destructuring can be used on parameters too
```suggestion
      {items.map(({ node }) => (
        <SearchResult node={node} profile={profile} />
```